src/ivoc/ocmatrix.h

```diff
@@ -139,13 +138,12 @@ class OcMatrix {
     }
 
     void unimp();
-    Object** temp_objvar();
 
   protected:
     OcMatrix(int type);
 
   public:
-    Object* obj_;
+    void* obj_;
```

Comment:
I don't understand the principles behind this change. It seems like ```Object*``` is a good indicator of what this is. And I thought ```void*``` was to be avoided if possible.

Reply:
I agree, I will revert this part ;)